tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecWorkflowTool/SpecWorkFlowTool.cs

```diff
@@ -452,14 +466,54 @@ public async Task<string> LinkSdkPullRequestToReleasePlan(string language, strin
                 }
 
                 await devopsService.AddSdkInfoInReleasePlanAsync(releasePlan.WorkItemId, language, "", parsedLink);
-                return $"Successfully linked pull request to release plan {releasePlan.ReleasePlanId}, work item id {releasePlan.WorkItemId}";
+
+                // Parse PR URL components and update PR description if valid
+                var prUrlMatch = GITHUB_PR_URL_REGEX.Match(parsedLink);
+                if (prUrlMatch.Success)
+                {
+                    var repoOwner = prUrlMatch.Groups[1].Value;
+                    var prNumber = int.Parse(prUrlMatch.Groups[3].Value);
+                    await AddReleasePlanInfoInSdkAsync(repoOwner, repoName, prNumber, releasePlan.ReleasePlanLink, releasePlan.ActiveSpecPullRequest, releasePlan.WorkItemUrl, releasePlan.SpecAPIVersion);
+                }
+
+                return $"Successfully linked pull request to release plan {releasePlan.ReleasePlanId}, work item id {releasePlan.WorkItemId}, and updated PR description.";
             }
             catch(Exception ex)
             {
                 SetFailure();
                 return $"Failed to link SDK pull request to release plan work item, Error: {ex.Message}";
             }
         }
+        
+        public async Task<string> AddReleasePlanInfoInSdkAsync(string repoOwner, string repoName, int prNumber, string releasePlanLink, string specPrLink, string workItemLink, string apiVersion)
+        {
+            var pr = await githubService.GetPullRequestAsync(repoOwner, repoName, prNumber);
+            if (pr == null)
+            {
+                return $"Failed to fetch pull request {repoOwner}/{repoName}#{prNumber}";
+            }
+            
+            // Check if the PR body already contains the release plan link (main indicator)
+            var header = "## Release Plan Details";
+            if (!string.IsNullOrEmpty(pr.Body) && pr.Body.Contains(header, StringComparison.OrdinalIgnoreCase))
+            {
+                return $"PR {repoOwner}/{repoName}#{prNumber} already contains release plan info. Skipping update.";
+            }
+
+            var links = $"{header}";
```

Comment:
I would use `StringBuilder` here with `.AppendLine` that way newlines are cross platform (\n vs \r\n). Also just a bit easier to construct/read.